src/components/Icon/index.js

```diff
@@ -41,27 +41,29 @@ class Icon extends PureComponent {
     render() {
         const width = this.props.small ? variables.iconSizeSmall : this.props.width;
         const height = this.props.small ? variables.iconSizeSmall : this.props.height;
-        const IconToRender = this.props.src;
-
-        const icon = (
-            <IconToRender
-                width={width}
-                height={height}
-                fill={this.props.fill}
-            />
-        );
 
         if (this.props.inline) {
             return (
-                <View style={[StyleUtils.getWidthAndHeightStyle(width, height), styles.bgTransparent, styles.overflowVisible]}>
+                <View
+                    accessibilityHint={`${this.props.src.name} Icon`}
```

Comment:
This small refactoring done in this file was to add this prop, which allowed the Icon to be targetted by the unit tests. The rest of the changes in this file are just some cleaning up.